paddle/fluid/operators/conv_op.h

```diff
@@ -27,6 +27,8 @@ namespace paddle {
 namespace operators {
 
 using Tensor = framework::Tensor;
+constexpr int kConvMKLDNNFP32 = 1;
+constexpr int kConvMKLDNNINT8 = 2;
```

Comment:
Maybe we need add an `attr` for every Op and the `attr`'s name is `algorithm`.
If so, we should also make a table to describe the meaning of every `algorithm`.

Reply:
No need to call it algorithm. Just make it customizable is enough